tools/aa/AaModule.java

```diff
@@ -68,4 +71,50 @@ public static String getCurrentWorkspaceName(
       throw new RuntimeException("You're not in a workspace");
     }
   }
+
+  private static final Integer NOT_A_VALUE = -1024;
```

Comment:
-1?
:)
Then you can avoid the `static final`.
I would document that the method returns -1 on invalid inputs and maybe rename to `safeParsePositiveInt`